compiler/src/scala/quoted/runtime/impl/QuoteMatcher.scala

```diff
@@ -121,19 +117,20 @@ object QuoteMatcher {
 
   private def withEnv[T](env: Env)(body: Env ?=> T): T = body(using env)
 
-  def treeMatch(scrutineeTree: Tree, patternTree: Tree)(using Context): Option[Tuple] =
+  def treeMatch(scrutineeTree: Tree, patternTree: Tree)(using Context): Option[Seq[Expr[Any]]] =
```

Comment:
Maybe make a type alias for Seq[Expr[Any]] and explain how it is used in a doc comment for it?

Reply:
I added `type MatchingExps` with its documentation.